python/box.tests/reports/git-history-plot.svg

```diff
@@ -62,39 +62,48 @@ text {
   <g class="plt-plot">
     <g transform="translate(20.0 10.0 ) ">
       <g transform="translate(42.5 451.0 ) " class="plt-axis">
-        <g class="tick" transform="translate(160.27933243119332 0.0 ) ">
```

Comment:
Looks like the MicroPython series didn't get an increase. It's because the plot is generated only for regular Python:

https://github.com/krzema12/kotlin-python/blob/af0c7cfbacc9fa1cc9bec5c8e68847b3946dc33a/.github/workflows/generate_reports.main.kts#L120

Merging independently generated plots for both pythons into one would mean some non-trivial effort. The good new is that with the next commit to `python-backend` branch, it should update the MicroPython series because the new failed-tests.txt for MP will be accessible from within the regular Python's reports generation task, so it's not a critical issue. Just pointing it out in case you want to retrigger it manually after merging, for the sake of keeping both series up-to-date :)

Reply:
I see the added commit is empty and nothing is updated in the plot.

Do you think it can be solved from the workflow side or should we change the generator for that?

Reply:
Good question, but I don't know and have no resources to investigate it today, sorry :( feel free to merge it and track it as a bug, to unblock us.